akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala

```diff
@@ -222,7 +222,14 @@ object FileAndResourceDirectives extends FileAndResourceDirectives {
             ResourceFile(url, entry.getSize, entry.getTime)
           }
         } finally jar.close()
-      case _ ⇒ None
+      case _ ⇒
+        val conn = url.openConnection()
```

Comment:
`url` is the result of `lassLoader.getResource` so it should point to a local resource. Dunno if that makes it safe though.